docs/usage.rst

```diff
@@ -452,6 +452,30 @@ functions that leverage this capability; builtins including, but not limited to:
 (for examples, see :ref:`sec_max_min`) and :func:`sorted`.
 
 
+Version Equality
+----------------
+
+Note that when comparing two versions, the ``build`` part is ignored::
```

Comment:
I think, we need to explain what _version equality_ is or how we understand that. Maybe something like this:

```suggestion
Version equality means for semver, that major, minor, and patch parts are
equal in both versions you compare. The build part is ignored, for example::
```

Maybe you can find a better phrase. :wink: